packages/browser/src/sdk.ts

```diff
@@ -97,6 +102,10 @@ export function init(options: BrowserOptions = {}): void {
     if (WINDOW.SENTRY_RELEASE && WINDOW.SENTRY_RELEASE.id) {
       options.release = WINDOW.SENTRY_RELEASE.id;
     }
+
+    if (typeof __SENTRY_RELEASE__ === 'string') {
```

Comment:
l: Should this take precedence over a release set on `window`? just thinking, to make sure this is what we expect :)

Reply:
Yes I think so. It's either that or the other way around.

Reply:
Hmm, you are more "into" these things, but I imagine it could be weird if you somehow set `window.SENTRY_RELEASE` to a specific value, just to have it be overwritten by __SENTRY_RELEASE__?

Reply:
I would put it like this: I would give precedence depending on how much work it is to set a value. To set `window.SENTRY_RELEASE` you just have to do exactly that - assign a value. To set `__SENTRY_RELEASE__` you have to instruct your bundler to replace that string, probably even by using some plugin. I would argue that this is a lot more effort to set this value.

I guess there is the argument that we want to have the value be configurable via window even if the bundler injects something. I can change it.

Reply:
> I guess there is the argument that we want to have the value be configurable via window even if the bundler injects something.

Yeah, that's basically my thought - _if_ someone overwrites this at runtime or whenever, I think it may make sense to have this take precendence. But I don't have strong feelings on this, so feel free to also keep it as it is if you think that makes more sense 👍

Reply:
No I def like it. Will change!